client/src/components/EventModal.vue

```diff
@@ -203,14 +240,57 @@
                 challengesF: [], challengesTextF: [],
                 challengesRE: [], challengesTextRE: [],
                 challengesC: [], challengesTextC: [],
-                cat: '', childrenChallenges: ''
+                cat: '', childrenChallenges: '', isDisabled: false,
+                disabledDates: {
+                    to: new Date(Date.now() - 8640000)
+                }
             }
         },
-        created: function(){
+        mounted: function(){
             this.getChallenges();
             this.getFrontends();
+            this.handleButtons();
         },
         methods: {
+            sendSlackNotification: function(isBooked, channel) {
+
+                if (this.handleSubmit()){
+                    return;
+                }
+
+                const Slack = require('slack')
+                const bot = new Slack();
+
+                const message = isBooked ? "*"+this.eventName + "* has been BOOKED" : "*"+this.eventName + "*has been CREATED";
+                const text = message + "\n" +
+                        "User: *" + localStorage.getItem("user-email") + "*\n" +
+                        "Event Tag: *" + this.eventTag + "*\n" +
+                        "Start Time: *" + this.get_date(this.eventStartTime) + "*\n" +
+                        "Finish Time: *" + this.get_date(this.eventFinishTime) + "*\n" +
+                        "Availability: *" + this.eventAvailability + "*\n" +
+                        "Capacity: *" + this.eventCapacity + "*\n" +
+                        "Frontend: *" + this.selectedFrontends + "*\n" +
+                        "Challenges: *" + this.selectedChallenges + "*";
+
+                window.console.log(process.env.SLACK_API_KEY);
```

Comment:
Could be nice if we remove it :) and change the name of  environment variable (SLACK_API_KEY) to something else, since it is leaked to github already.

Reply:
SLACK_API_KEY does not work.

Reply:
This library could be useful to use to get SLACK running, https://github.com/slackapi/node-slack-sdk